stable/jenkins/values.yaml

```diff
@@ -81,3 +82,10 @@ Persistence:
 #  - mountPath: /var/nothing
 #    name: nothing
 #    readOnly: true
+
+NetworkPolicy:
+  # Enable creation of NetworkPolicy resources.
+  Enabled: true
```

Comment:
The only difference is that CI can't really test the changes if it could detect a problem anyway. I would bump this to 0.7.x chart version if that's the case.